Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveService.cs

```diff
@@ -326,5 +330,50 @@ public bool Initialize(string appClientId, AccountProviderType accountProviderTy
             var oneDriveRootItem = await _oneDriveProvider.Drive.Special.Photos.Request().GetAsync();
             return new OneDriveStorageFolder(_oneDriveProvider, _oneDriveProvider.Drive.Special.Photos, oneDriveRootItem);
         }
+
+        /// <summary>
+        /// Creates a background download for given OneDriveId
+        /// </summary>
+        /// <param name="oneDriveId">OneDrive item's Id</param>
+        /// <param name="destinationFile">A <see cref="StorageFile"/> to which content will be downloaded</param>
+        /// <param name="completionGroup">The <see cref="BackgroundTransferCompletionGroup"/> to which should <see cref="BackgroundDownloader"/> reffer to</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the request</param>
+        /// <returns>The created <see cref="DownloadOperation"/></returns>
+        public async Task<DownloadOperation> CreateBackgroundDownloadForItemAsync(string oneDriveId, StorageFile destinationFile, BackgroundTransferCompletionGroup completionGroup = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (string.IsNullOrWhiteSpace(oneDriveId))
+            {
+                throw new ArgumentNullException(nameof(oneDriveId));
+            }
+
+            if (destinationFile == null)
+            {
+                throw new ArgumentNullException(nameof(destinationFile));
+            }
+
+            // log the user silently with a Microsoft Account associate to Windows
+            if (_isConnected == false)
+            {
+                OneDriveService.Instance.Initialize();
+                if (!await OneDriveService.Instance.LoginAsync())
+                {
+                    throw new Exception("Unable to sign in");
+                }
+            }
+
+            return await Task.Run(
+                async () =>
+                {
+                    var requestMessage = Provider.Drive.Items[oneDriveId].Content.Request().GetHttpRequestMessage();
+                    await Provider.AuthenticationProvider.AuthenticateRequestAsync(requestMessage).AsAsyncAction().AsTask(cancellationToken);
```

Comment:
@PedroLamas I wonder if we need those cancellation tokens there, as this part of code runs on separate thread which has it's own cancellation token *Task.Run(Action, cancellationToken)*. 

I would like to know how this behaves in case od cancel signal, the thread is ended instantly/methods interrupted? Though I don't know where I can find such information and don't have much time now to go across reference source.